onnxmltools/convert/keras/shape_calculators/GlobalPooling.py

```diff
@@ -0,0 +1,23 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import keras
+from ...common._registration import register_shape_calculator
+
+def calculate_keras_global_pooling_output_shapes(operator):
```

Comment:
The original shape should not be changed. We should add an Reshape into its converter instead.

Reply:
yeah, you are right, I am working on it.

Reply:
fixed.